src/MongoDB.Driver.Core/Core/WireProtocol/CommandUsingCommandMessageWireProtocol.cs

```diff
@@ -190,7 +191,10 @@ private Type0CommandMessageSection<BsonDocument> CreateType0Section(ConnectionDe
             var dbElement = new BsonElement("$db", _databaseNamespace.DatabaseName);
             extraElements.Add(dbElement);
 
-            if (_readPreference != null && _readPreference != ReadPreference.Primary)
+            // if standalone, don't send read preference
```

Comment:
I find it more confusing with the comment than without, because the comment only applies to the first of the three clauses in the conditional, and anyway it just repeats what the first clause says already quite clearly.

I suggest the comment just be removed.

Reply:
Done